spec/system/school/courses_index_spec.rb

```diff
@@ -101,12 +101,12 @@ def file_path(filename)
       fill_in 'Course name',
               with: new_course_name,
               fill_options: {
-                clear: :backspace
+                clear: :backspace,
               }
       fill_in 'Course description',
               with: new_description,
               fill_options: {
-                clear: :backspace
+                clear: :backspace,
```

Comment:
@prashanth726 I think your Ruby formatting is not set up properly. We are currently formatting using the prettier-ruby plugin in the project. https://github.com/prettier/plugin-ruby#usage-with-an-editor